portfolio/src/main/webapp/index.html

```diff
@@ -5,11 +5,13 @@
     <title>Cesar Rivera</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+   <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC7qtzw26sbO1utXi2GSx1JnwTBNILlbfU"></script>
```

Comment:
I think this is the API key that shouldn't be committed.

Reply:
Just took the API key out and commit it to the branch, still trying to see how I can tell git to ignore DataServlet.java

Reply:
For the DataServlet.java part, my guess is that your git tree looks something like this:
PR8 -> PR9
       \-> PR10 -> PR11
(where PRn means "the chain of commits used to generate that PR). I think you'd want to rebase PR10 so that it's based on `upstream/master` (which is currently PR9).